app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailDestination.kt

```diff
@@ -15,13 +15,14 @@ fun NavGraphBuilder.detailGraph(
     navigateToTopicList: () -> Unit
 ) {
     composable(
-        route = DetailDestination.route + "?level={level}&id={id}&tag={tag}&topic={topic}&shareLink={shareLink}",
+        route = DetailDestination.route + "?level={level}&id={id}&tag={tag}&topic={topic}&shareLink={shareLink}&bgColor={bgColor}",
```

Comment:
🚫 **[ktlint]** <max-line-length> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Exceeded max line length (100) (cannot be auto-corrected)